wix-bookings-v2/wix-bookings-v2/AvailabilityCalendar/guides/Supported Filters and Sorting.md

```diff
@@ -0,0 +1,60 @@
+| `bookable`            |  |  | When filtered by `true`, returns only available slots. Otherwise, returns both available and non-available slots.                                                                                                                                                                                                      |
+| `bookingPolicyViolations.tooEarlyToBook` |  |  |                                                                                                                                                                                                                                                                                                                        |
+| `bookingPolicyViolations.tooLateToBook` |  |  |                                                                                                                                                                                                                                                                                                                        |
+| `location.businessLocation.id`  |  |  | Supports multiple values. Currently, you can query for multiple business location IDs but this functionality will change in the near future. At that point, multiple locations will not be supported for appointments. For appointments, you must provide only 1 business location ID as part of any query you run. We recommend you begin to modify your code accordingly. |
+| `openSpots` |  |  | Returns slots with at least this number of open spots.                                                                                                                                                                                                                                                                 |
+| `createdDate`   |  |  |                                                                                                                                                                                                                                                                                                                        |
+| `updatedDate`   |  |  |                                                                                                                                                                                                                                                                                                                        |
+
+
+## Sorting 
+
+Currently, only sorting by `startDate` is supported.
+
+Specify the `sort` object in the following format:  
+
+```json
+"sort" : { 
+    "fieldName":"sortField1",
+    "order":"ASC"
+  }
+```
+
+
+__Related content:__
+[API Query Language](https://www.wix.com/velo/reference/api-overview/api-query-language),
+[queryAvailability( )](https://www.wix.com/velo/reference/wix-bookings-v2/availabilitycalendar/queryavailability)
```

Comment:
@christopherbWIX   Hot off the presses! Fernando just merged yesterday a NEW version of this article. Please substitute this version for that version.